styles.css

```diff
@@ -0,0 +1,109 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  max-width: 375px;
+  font-family: "Poppins", sans-serif;
+  margin: 0 auto;
+  background: #fff;
+}
+
+header {
+  background-color: #fff;
+  max-width: 375px;
+  height: 20px;
+  padding-top: 28 px;
+}
+
+.container1 {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #6070ff;
+  margin-top: 20px;
+}
+
+.logo1 {
+  color: #6070ff;
+  text-decoration: none;
+  font-family: "Poppins", sans-serif;
+  line-height: 20px;
+  font-weight: 700;
+  font-size: 20px;
+  padding-left: 25px;
+  padding-top: -30px;
+}
+
+.menu-bar {
+  padding-top: 10px;
+  padding-right: 20px;
+  font-size: 25px;
+}
+
+.sadat {
+  margin-top: 10%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  /* height: 100vh; */
```

Comment:
- [OPTIONAL] I would like to recommend you not too leave unused line of code in your codebase. It makes your code looks messy. I decided not to require changes because of this little issue, please, treat it as very important practice not the leave commentted code in your codebase in your future projects.